server/services/graph-api.js

```diff
@@ -0,0 +1,25 @@
+const request = require('request');
+
+const bunyan = require('bunyan');
+
+const log = bunyan.createLogger({ name: 'graph-api-service' });
+
+module.exports = class GraphAPi {
+  static callSendAPI(requestBody) {
+    request(
+      {
+        uri: 'https://graph.facebook.com/v3.2/me/messages',
+        qs: { access_token: process.env.PAGE_ACCESS_TOKEN },
+        method: 'POST',
+        json: requestBody,
+      },
+      (error) => {
+        if (error) {
+          log.error('Unable to send message:', error);
+        } else {
+          log.info('message sent!');
+        }
+      },
+    );
+  }
+};
```

Comment:
Are there any other use cases for the class GraphAPI? If not, since you only have one static function, you can just export the function itself instead of creating the GraphAPI class

Reply:
In the future, we could add profiling and sending profile would be easily addable from here. We could just remove it if you foresee that profiling would not be added until the far future.

Reply:
In that case, I think we can leave it that way for now